src/main/kotlin/frc/robot/commands/DriveDoubleSupplier.kt

```diff
@@ -25,7 +25,9 @@ class DriveDoubleSupplier(val driveSubsystem: DrivetrainSubsystem, val forwardSp
 
     override fun end(interrupted: Boolean) {
         driveSubsystem.driveArcade(0.0, 0.0, true)
+        // stops drivetrain by sending 0 speeds
```

Comment:
Placing comments after the code they describe makes it hard to see what code they are referring to. Comments are nearly always placed above or alongside code. Any of the following would be acceptable:
```
override fun end(interrupted: Boolean) {
    // stops drivetrain by sending 0 speeds
    driveSubsystem.driveArcade(0.0, 0.0, true)
}
```
```
// stops drivetrain by sending 0 speeds
override fun end(interrupted: Boolean) {
    driveSubsystem.driveArcade(0.0, 0.0, true)
}
```

```
override fun end(interrupted: Boolean) {
    driveSubsystem.driveArcade(0.0, 0.0, true)  // stops drivetrain by sending 0 speeds
}
```